### DISCUSSION TITLE
Vite + PostCSS with react Swagger UI

### DISCUSSION THREAD

**Discussion by bastiengrignon** [swagger-api/swagger-ui#10088] [Q&A]
I've added `postcss-simple-vars` to my dependencies and now  I have an error in this file `/swagger-ui-react/swagger-ui.css:2:2` which is **Undefined variable $vocabulary** ! 

How can I keep using swaggerUI with postCSS config ?

**Comment by vavilov2212** [comment]
https://github.com/swagger-api/swagger-ui/issues/10045
